public/widget/discrepancy-note/Dn.js

```diff
@@ -583,7 +583,7 @@ Comment.prototype._renderHistory = function() {
     var $more = over3 > 0 ? $( '<tr><td colspan="4"><span class="over">+' + over3 + '</span>' +
         '<button class="btn-icon-only btn-more-history"><i class="icon"> </i></button></td></tr>' ) : $();
     this.$history.find( 'table' ).empty()
-        .append( '<thead><tr><td></td><td>' + historyText + '</td><td>' + user + '</td><td>' + clock + '</td></tr></thead>' )
+        .append( '<thead><tr><td colspan="2"><b>' + historyText + '</b></td><td>' + user + '</td><td>' + clock + '</td></tr></thead>' )
```

Comment:
Removing the second column means that multi-line comments are not indented. 
<img width="343" alt="screen shot 2017-06-07 at 4 20 13 pm" src="https://user-images.githubusercontent.com/627350/26904036-5a794e7a-4b9d-11e7-8fa3-52d009d129ce.png">

It's a matter of preference preference perhaps. I'd prefer to keep them in separate columns for that reason (and just merge the 2 header columns).